docs/model_configuration.md

```diff
@@ -125,9 +125,11 @@ When a model is using the auto-complete feature, a default maximum batch size ma
 be set by using the `--backend-config=default-max-batch-size=<int>` command line argument. 
 This allows all models which are capable of batching and which make use of 
 [Auto Generated Model configuration](#auto-generated-model-configuration) to have
-a default maximum batch size. This value is set to 4 by default. While none of the
-officially supported Triton backends implement this feature, backend developers 
-may make use of this value by obtaining it from the TRITONBACKEND_BackendConfig api.
+a default maximum batch size. This value is set to 2 by default. If the value is
+greater than 1, the dynamic batch scheduler will be set for the model if
+no scheduler is provided in the configuration file. Backends which support this 
```

Comment:
Currently, the following backends utilizes these default batch values and enable dynamic batching in generated model configuration:

1. TensorFlow backend
2. OnnxRuntime backend

Add link to the backend repo ^

Reply:
Added the statement with links to the backends.